homeassistant/components/numato/binary_sensor.py

```diff
@@ -120,11 +120,6 @@ def _async_update_state(self, level):
         self._state = level
         self.async_write_ha_state()
 
-    @property
-    def name(self):
-        """Return the name of the sensor."""
-        return self._name
-
     @property
     def is_on(self):
```

Comment:
Note: `is_on` has some special logic which needs special adjustments before migrating to shorthand attribute.